cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -534,7 +540,15 @@ private ApiClusterTemplate getCmTemplate(TemplatePreparationObject templatePrepa
 
     private void configureCmMgmtServices(TemplatePreparationObject templatePreparationObject, String sdxCrn, Telemetry telemetry,
             String sdxContextName, ProxyConfig proxyConfig) throws ApiException {
-        Optional<ApiHost> optionalCmHost = getCmHost(templatePreparationObject, apiClient);
+        Optional<ApiHost> optionalCmHost;
+        if (null != templatePreparationObject.getStackType()
+            && templatePreparationObject.getStackType().equals(StackType.DATALAKE)
+            && isVersionNewerOrEqualThanLimited(templatePreparationObject.getBlueprintView().getVersion(), CLOUDERA_STACK_VERSION_7_2_17)
+            && containsIgnoreCase(templatePreparationObject.getBlueprintView().getBlueprintText(), SDX_ENTERPRISE_DATALAKE_TEXT)) {
+            optionalCmHost = getAuxiliaryHost(templatePreparationObject, apiClient);
```

Comment:
Instead of looking for a host group by name, it would be ideal to find a host group by instance count.
We want to move CM MGMT services to a host group with a single node. Name chance to the instance group will break this